build.cake

```diff
@@ -233,6 +235,32 @@ Task("Create-Tools-Package")
             }
         });
     });
+
+Task("Create-Installer")
+    .IsDependentOn("Copy-Files")
+    .Does(() => {
+        CopyDirectory(binDir, installerDir + Directory("lib") + Directory("net45"));
+        CopyFileToDirectory("./src/Wyam.nuspec", installerDir);
+        var nuspec = GetFiles(installerDir + File("Wyam.nuspec")).FirstOrDefault();
+
+        NuGetPack(nuspec, new NuGetPackSettings
```

Comment:
I don't think so. The one that is packed here contains everything that would otherwise be distributed in the zip / bin directory. The one you are referring to (but maybe I got that wrong) is referencing the files from the `bin` folder, but as they are not placed in `lib/net45` Squirrel won't accept them. But potentially I've missed something. Having one package would be great, but I did not want to disrupt your current setup too much.